component/Card/style.css

```diff
@@ -0,0 +1,196 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body {
+  font-family:san-serif;
+  background-color: rgb(151, 151, 151);
+  font-family: sans-serif;
```

Comment:
font-family property is mentioned two times

Reply:
resolve